futures/src/lib.rs

```diff
@@ -228,31 +228,13 @@ pub mod prelude {
     //!
     //! The prelude may grow over time as additional items see ubiquitous use.
 
-    pub use futures_core::future::{Future, TryFuture};
-    pub use futures_core::stream::{Stream, TryStream};
-    pub use futures_core::task::{self, Poll};
-
-    pub use futures_sink::Sink;
+    pub use crate::future::{self, Future, TryFuture, FutureExt, TryFutureExt};
+    pub use crate::stream::{self, Stream, TryStream, StreamExt, TryStreamExt};
+    pub use crate::task::{self, Poll, ContextExt};
+    pub use crate::sink::{self, Sink, SinkExt};
```

Comment:
The prelude now additionally exports `future`, `stream` and `sink`

Reply:
Nice!